images/busybox/alpine.tf

```diff
@@ -0,0 +1,32 @@
+module "latest-alpine" {
+  providers = {
+    apko = apko.alpine
+  }
```

Comment:
I tried to do something similar with `static` [here](https://github.com/chainguard-images/images/pull/958), but locally I'm seeing errors which make me question whether this is WAI 🤔 

The alpine build is trying to find the `wolfi-baselayout` package and can't because that's not one of the repositories: 
```
│ Error: error computing package locks
│ 
│   with module.static.module.alpine.module.this.data.apko_config.this,
│   on .terraform/modules/static.alpine.this/main.tf line 17, in data "apko_config" "this":
│   17: data "apko_config" "this" {
│ 
│ resolving apk packages: could not find package, alias or a package that provides wolfi-baselayout in indexes
```